hack/user_setup_env.sh

```
@@ -0,0 +1,116 @@
+}
+
+install_krew() {
+    log "Installing krew"
+
+    KREW="krew-${OS}_${ARCH}"
+    curl -o "/tmp/${KREW}.tar.gz" -fsLO "https://github.com/kubernetes-sigs/krew/releases/download/${krew_version}/${KREW}.tar.gz"
+    mkdir /tmp/krew && tar -xzf "/tmp/${KREW}.tar.gz" -C /tmp/krew/
+    "/tmp/krew/${KREW}" install krew
+    log "configure .bashrc for krew"
+    printf '# krew\nexport PATH="${KREW_ROOT:-$HOME/.krew}/bin:$PATH"\n' >> ~/.bashrc
+    source ~/.bashrc
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)